### ISSUE TITLE
Function bind operator fails `no-invalid-this` rule

### ISSUE THREAD

**Issue by vinsonchuong** [babel/babel-eslint#362]
```js
function fn() {
  return this.value;
}

const obj = {
  value: 42
};

obj::fn();
```

I can see why `eslint` proper doesn't support this usecase. Is it possible for `babel-eslint` to provide a `babel/no-invalid-this` to support this usecase?

**Comment by hzoo** [comment]
If you need a rule/modify one you can bring it up as an issue/pr at https://github.com/babel/eslint-plugin-babel

**Comment by vinsonchuong** [comment]
Ok, moving to babel/eslint-plugin-babel#82
